README.md

```diff
@@ -90,6 +90,41 @@ in node:
 
 Feel free to [try immutad●t on runkit](https://npm.runkit.com/immutadot).
 
+## Dot notation
+
+immutad●t brings a few improvements to the classic dot notation:
+
+### Slice notation
+
+The slice notation let you go trough arrays to apply operations without having to map arrays at each level of imbrication.
+
+```js
+add({ nested: { prop: [{ val: 1 }, { val: 2 }] } }, 'nested.prop[:].val', 1)
+// → { nested: { prop: [{ val: 2 }, { val: 3}] } }
+
+divide({ nested: { prop: [{ val: 1 }, { val: 2 }] } }, 'nested.prop[1:].val', 2)
+// → { nested: { prop: [{ val: 1 }, { val: 1}] } }
+
+multiply({ nested: { prop: [{ val: 1 }, { val: 2 }, { val: 3 }, { val: 4 }] } }, 'nested.prop[1:2].val', 2)
+// → { nested: { prop: [{ val: 1 }, { val: 4 }, { val: 6 }, { val: 4 }] } }
+```
+
+### List notation
+
+The list notation let you go trough each keys of objects used as collection or map to apply operations.
```

Comment:
What about "The list notation lets you iterate over the keys..."
It's not necessarily each keys...